PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -980,68 +970,23 @@ void PATMuonProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
       muon.setSelector(reco::Muon::PuppiIsoTight, puppiCombinedIsolationPAT < 0.12);
     }
 
-    float jetPtRatio = 0.0;
-    float jetPtRel = 0.0;
-    float mva = 0.0;
-    float mva_lowpt = 0.0;
-    if (computeMuonMVA_ && primaryVertexIsValid && computeMiniIso_) {
+    std::vector<float> jetPtRatioRel = {0.0, 0.0};
+    if (primaryVertexIsValid && computeMiniIso_) {
       if (mvaUseJec_) {
```

Comment:
Could you please rename `mvaUseJec_`? It can be confusing considering that MVAs aren't computed anymore